### DISCUSSION TITLE
the structure parametres of the sys_configs ,eg.gaussian

### DISCUSSION THREAD

**Discussion by maoxinxina** [deepmodeling/dpgen#1162] [General]
here I am using the dpgen-gaussian,howeve, I don't know the style which i should offer in the keyword of sys_configs

**Comment by njzjz** [comment]
The style is set in the input file:

https://docs.deepmodeling.com/projects/dpgen/en/latest/run/param.html#argument:run_jdata/sys_format

**Reply by maoxinxina** [reply]
Thanks, this refers to file types like POSCAR or gaussian/log?
